packages/elements/src/photon-advanced-medication-search-dialog/photon-advanced-medication-search-dialog-component.tsx

```diff
@@ -88,7 +88,11 @@ const Component = (props: MedSearchDialogProps) => {
         on:photon-dialog-canceled={handleCancel}
         on:photon-dialog-alt={handleCancel}
       >
-        <photon-med-search open={props.open} with-concept={props.withConcept} title={props.title} />
+        <photon-advanced-medication-search
```

Comment:
Is advanced search the old advancaed search?

Reply:
yes